.evergreen/run-tests.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
What is `run-tests` for? I don't see where it's referenced locally, or where this script comes from.

Reply:
If we are not actively running run-tests.sh in our Evergreen config, we'll only need to update it when we add a new branch to the continuous matrix testing project: https://github.com/mongodb/drivers-matrix-testing/blob/2b5d4e388420305adf10eb4d42824da10ae84c2e/.evergreen/config.yml#L588

I would argue if we can't use `run-tests.sh` actively, there's not much value in adding it now.

@divjotarora was the intent behind the ticket to have `run-tests.sh` be the one way we run tests? I.e. update config.yml's "run-tests" function to use it?